nx_parallel/algorithms/centrality/tests/test_betweenness_centrality.py

```diff
@@ -3,7 +3,7 @@
 import math
 
 
-def test_betweenness_centrality_get_chunks():
+def get_chunk_function(ebc):
     def get_chunk(nodes):
```

Comment:
you can put this `get_chunk` back in the `test_betweenness_centrality_get_chunks` as it is not used anywhere else.

Reply:
done